tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/errors/FunctionNotFoundError.java

```diff
@@ -0,0 +1,35 @@
+package com.google.cloud.bigquery.utils.queryfixer.errors;
+
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.utils.queryfixer.entity.Position;
+
+import lombok.Getter;
+
+/** A class to represent the "Function Not Found" errors. The errors look like one of the two forms: (1) Function not
```

Comment:
nit: the auto formatting is really weird... :( For multiple options (1) and (2) just use new line

Reply:
I think auto formatting does not handle it. Are you suggesting to put (1) and (2) in two different lines?